doc/updating_documentation/How_to_update_this_documentation.md

```diff
@@ -0,0 +1,28 @@
+# How to update this documentation
+
+## How does it work
+
+Documentation is stored on master branch of Juniper/contrail-windows-docs repository. It is stored in Markdown (.md) format. Documentation is served as HTML on github.io. HTML pages are generated by MkDocs tool. HTML pages need to be redeployed manually after modification.
+
+## Documentation update procedure
+
+1. Open a pull request to master branch in repository Juniper/contrail-windows-docs with your changes.
+2. Once PR is accepted, an Administrator is going to deploy new modifications to gh-pages branch.
+
+## How to preview modifications
+
+1. Install MkDocs (see below).
+2. Apply required modifications to the documentation.
+3. Invoke `mkdocs serve` in main repository directory. It's going to start local HTTP server with documentation preview. Follow instructions displayed by mkdocs to see the outcome.
+
+## How to install MkDocs
+
+MkDocs requires Python (both Python 2 and Python 3 are supported) and PIP. Follow instructions on [http://www.mkdocs.org/#installation](http://www.mkdocs.org/#installation). Basically `pip install mkdocs` should work. It may also be useful to use virtualenv to keep mkdocs and its dependencies in one isolated place.
+
+## How to deploy modifications
+
+After documentation is modified, github.io pages need to be redeployed manually. To do this, follow these steps:
```

Comment:
> After documentation is modified

seems ambiguous. Maybe "After master branch of Juniper/contrail-windows-docs repository is updated, github.io pages need to be redeployed manually"?

Reply:
This is exactly what you suggested... I'll change it.

Reply:
ᕕ( ᐛ )ᕗ

You're right, sorry. Redaction is an iterative process though.